### DISCUSSION TITLE
Query on data sizing

### DISCUSSION THREAD

**Discussion by ankitsoin** [finos/TimeBase-CE#271] [General]
HI @alex-karpovich ,

As mentioned in https://kb.timebase.info/community/overview/streams#data-sizing 
5.x Storage Format - 6.8 Mb/1 million messages
4.x Storage Format  - 32 Mb/1 million messages

So storage format is making great impact in terms of space used, but couldn't able to find where we have to define the same. Please help us in same.
Also, are there any imitations of using 5.X format?

**Comment by alex-karpovich** [comment]
@ankitsoin TimeBase-CE is using 5.0 format

**Reply by ankitsoin** [reply]
Great, thanks @alex-karpovich for the reply.
